go/mysql/server.go

```diff
@@ -55,6 +55,7 @@ var (
 	timings    = stats.NewTimings("MysqlServerTimings", "MySQL server timings", "operation")
 	connCount  = stats.NewGauge("MysqlServerConnCount", "Active MySQL server connections")
 	connAccept = stats.NewCounter("MysqlServerConnAccepted", "Connections accepted by MySQL server")
+	connRefuse = stats.NewCounter("MysqlServerConnRefused", "Connections closed by MySQL server")
```

Comment:
Also fix description of counter:
closed -> refused

Reply:
Good catch - pushing a change for this now